ogr/abstract.py

```diff
@@ -157,14 +133,24 @@ def __init__(
     def __str__(self) -> str:
         comment = f"{self.comment[:10]}..." if self.comment is not None else "None"
         return (
-            f"PRComment("
+            f"Comment("
             f"comment='{comment}', "
             f"author='{self.author}', "
             f"created='{self.created}', "
             f"edited='{self.edited}')"
         )
 
 
+class IssueComment(Comment):
+    def __str__(self) -> str:
+        return "Issue" + super().__str__()
```

Comment:
What about the backward link to the original issue object?

Reply:
added TODO in services' `comments.py`